lib/store/util/vuex_resource.js

```diff
@@ -145,6 +144,34 @@ export function generateCrud(model) {
       })
   }
 
+  actions[copyModelName] = ({ dispatch, commit, getters }, existingModel) => {
+    debug("Action:", copyModelName, existingModel)
+    let modifiedExistingModel = {
+      ...existingModel,
+      name: "Copy of " + existingModel.name
+    }
+    let newModel = model.create(modifiedExistingModel)
+
+    let relationshipsToInitialize = filter(model.relationships, (relation) => relation.initialize)
```

Comment:
This looks good, looks like it will work for copying most other resources as well.

Except Templates, but they are a special beast because they have to copy every one of their layers as well. (I think.) This won't see them because the layers relationship on template doesn't use `initialize: true`, but we'll figure it out.

Reply:
Yes, will figure out for all types of relationships what is the more adequate copying type. So far it's only deep copying, but I'll look at the other ones, and the special case for layers.